### PR TITLE
Middle section mobile website

### PR DESCRIPTION
**Pull request by VaheStepanyan100**
-  Parse a Figma design to create a UI.
- Use Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.
- Build a personal portfolio site.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -58,18 +63,207 @@
...
+
+.work {
+  width: 327px;
+  height: 578px;
+  margin-left: 24px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+}
```
Comment (levy002) [inline styles.css:98]: - [ ] KIndly make sure each project card content is aligned on the page using the CSS `flexbox` in order to meet the project requirements. Not only that, CSS `flexbox` will help in the future to maintain easily page responsiveness. Kindly read [this lesson](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox) for more information.

- [ ] KIndly make sure to give the project card the same `background-color` as the one on the provided Figma design.

    - ### YOUR DESIGN
    
    image

    - ### FIGMA DESIGN
    
    image

File: index.html [hunk 2]
```diff
@@ -30,26 +30,172 @@ <h1>
...
+      <section class="about-section">
+        <article>
+          <h2>About Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <div class="contactTitle"><a href="#">LET’S CONNECT</a></div>
```
Comment (levy002) [inline index.html:151]: - [ ] KIndly make sure the `LET'S CONNECT` title is rendered on the page using heading tags `h2, h3, h4, hx. ..` in order to follow best HTML practices and meet the project's requirements a well.

File: index.html [hunk 3]
```diff
@@ -30,26 +30,172 @@ <h1>
...
+          </ul>
+          <div class="frame-title">
+            <span>Frameworks</span>
+            <img src="assets/Union2.png" alt="" />
+          </div>
+          <hr />
+          <div class="skill-title">
+            <span>Skills</span>
+            <img src="assets/Union2.png" alt="" />
+          </div>
```
Comment (levy002) [inline index.html:196]: - [ ] KIndly make sure `Languages`, `Frameworks`, and `Skills` are wrapped into the `ul` tag and each one is wrapped into the `li` tag in order to follow best HTML practices and meet the project's requirements a well.

File: styles.css [hunk 4]
```diff
@@ -16,6 +17,10 @@
   text-decoration: none;
 }
 
+main {
+  background-color: #f7f7f9;
+}
+
```
Comment (levy002) [inline styles.css:23]: - [ ] Kindly make sure your `main` section covers all widths of the page. You can achieve that by making sure the body doesn't have any `padding` or `margin`.

File: styles.css [hunk 5]
```diff
@@ -58,18 +63,207 @@
...
+  flex-direction: column;
+  gap: 16px;
+}
+
+.languages > li {
+  background-color: #f7f7f9;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+}
```
Comment (levy002) [inline styles.css:243]: - [ ] Kindly make sure to add the padding on all sides of the language card in order to meet the provided Figma design.

   ### YOUR DESIGN
    image


   ### FIGMA DESIGN
    image

File: index.html [hunk 6]
```diff
@@ -30,26 +30,172 @@ <h1>
...
+        <article>
+          <h2>About Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <div class="contactTitle"><a href="#">LET’S CONNECT</a></div>
+          <ul class="lets-connect">
```
Comment (levy002) [inline index.html:152]: - [ ] KIndly make sure to provide enough space between this title and the social media links container in order to look like the provided Figma design as much as possible.


    - ### YOUR DESIGN
    
    image

    - ### FIGMA DESIGN
    
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @VaheStepanyan100   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Professional PR :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -30,26 +30,180 @@ <h1>
...
+            <li>
+              <div class="frame-title">
+                <span>Frameworks</span>
+                <img src="assets/Union2.png" alt="" />
+              </div>
+              <hr />
+            </li>
+            <li>
+              <div class="skill-title">
+                <span>Skills</span>
```
Comment (OyePriscilla) [inline index.html:200]: - [ ] Please kindly wrap the list of the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li` as suggested earlier. 👍 

- [optional] For good user interaction, kindly design this technologies categories according to the template. 👍 

## Your design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @VaheStepanyan100 👍 👍 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Displayed the project section with CSS flex property ✔️
Used images and background-image to enhance the look of the website ✔️
Made some required change ✔️


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @VaheStepanyan100 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Displayed the project section with CSS flex property ✔️
Used images and background-image to enhance the look of the website ✔️
Made some required change ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**